neqo-transport/tests/network.rs

```diff
@@ -148,6 +149,147 @@ simulate!(
     ],
 );
 
+// bandwidth 10Mbps, 20Mbps, 50Mbps and 100Mbps
+// delay by 10ms, 50ms, 100ms and 250ms
+// loss 1%, 2%, 5%, 10% and 20%
+// With tail-drop queue
+#[test]
+fn diff_conditions_client_download() {
```

Comment:
How much do you think that client download and server upload will be different?  The server can send earlier than a client, but that's all I've got.

Reply:
Yes, they are the same, I should not upload the fist version of the patch, it is never good. I will upload an update soon.